scripts/rollup/bundles.js

```diff
@@ -465,7 +465,7 @@ const bundles = [
     moduleType: NON_FIBER_RENDERER,
     entry: 'react-events/press',
     global: 'ReactEventsPress',
-    externals: ['react'],
+    externals: ['react-dom'],
```

Comment:
I believe this means that we should add it as a dep in the package.json.

https://github.com/facebook/react/blob/master/packages/react-events/package.json#L40

I  know yarn pnp relied on this package information for module resolution.

That's unfortunate since it's not a dep for the whole package. Just when used in a DOM environment. Not sure how mixed RN / web code deals with this. cc @necolas

Reply:
This isn't a big deal for now. We aren't going to be publishing `react-events` anytime soon. This was one of the reasons I tried to avoid this approach to begin with, but we could get around this with some kind of build step, it just means that these can't really ever go in `react`, but must be their own separate package – otherwise `react` will pull in ever renderer out there.